README.md

```diff
@@ -0,0 +1,66 @@
+# `${dataset_name}`
+
+Part of [`spinalcordtoolbox`](https://github.com/spinalcordtoolbox).
+
+
+## Using the Template
+
+1. Make a [new repo](https://github.com/new) using this as its template.
+
+    Name it `spinalcordtoolbox/data-${dataset_name}` where `${dataset_name}` is something usefully descriptive:
+
+    ![data-template-new](./.data-template-new.png)
+
+3. Download the new repo
+    
+    ```
+    git clone git@github.com:spinalcordtoolbox/data-${dataset_name}
+    cd data-${dataset_name}
+    ```
+    
+1. Fill with initial metadata
+    
+    ```
+    git mv src/spinalcordtoolbox/data/dataset src/spinalcordtoolbox/data/${dataset-name}
+    vi README.md # 1. remove this 'Using the Template' section
+                 # 2. change remaining ${dataset_name}s to the name you picked
+    vi setup.py  # 1. set name=spinalcordtoolbox-data-${dataset_name}
+                 # 2. set url=https://github.com/spinalcordtoolbox/data-${dataset_name}
+    git add -u
+    git commit -m "Initial commit"
+    ```
+    
+1. Fill with initial data and upload
+    
+    ```
+    cp ${data_files} src/spinalcordtoolbox/data/${dataset_name} 
+    git add .
+    git commit
+    git push
+    ```
+
+
+## How to update this dataset
+
+1. Edit and commit the files in `src/spinalcordtoolbox/data/`
+2. Go to https://github.com/spinalcordtoolbox/data-${dataset_name}/releases
+
+    1. Click "Draft Release"
+    2. Fill in a version tag. We use [this versioning policy](TODO)
```

Comment:
I'm partial to versioning as `1`, `2`, `3`, ... . It's dead easy, compliant with PEP 440 (`[N!]N(.N)*[{a|b|rc}N][.postN][.devN]`), and I don't really see how we'd end up with "major" and "minor" versions of our datasets. I suppose that if we want to be fancy, we can also support appending `.devN`.